### PR TITLE
Add test for vai91

### LINKED ISSUE #12: Test with vai91
Description....

### PR DESCRIPTION
**Pull request by TheCandiko**
Closes #12

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world! for vai91!
```
Comment (efeekiner) [inline test.txt:1]: Review was needed, here it is!

Comment (TheCandiko) [reply test.txt:1]: Ohh thanks maan!!!

Comment (vai91) [reply test.txt]: Inline comment

Comment (efeekiner) [reply test.txt:1]: Inline comment

**Review by efeekiner — COMMENTED** [review]
review summary

File: test.txt (hunk 1 above)
Comment (efeekiner) [reply test.txt:1]: Comment again

**Review by efeekiner — APPROVED** [review]
summary of the review

**Pull request merged**